scripts/expert/Botania.zs

```diff
@@ -65,16 +65,16 @@ mods.thaumcraft.Infusion.registerRecipe("GaiaPylon", "", <botania:pylon:2>, 6, [
 
 # -Catalysts
 recipes.remove(<botania:alchemycatalyst>);
-recipes.addShaped(<botania:alchemycatalyst>, [[<ore:livingrock>, <bewitchment:focal_chalk>, <ore:livingrock>], [<bloodmagic:slate:1>, <thaumcraft:salis_mundus>, <bloodmagic:slate:1>], [<ore:livingrock>, <ore:blockblaze>, <ore:livingrock>]]);
+recipes.addShaped(<botania:alchemycatalyst>, [[<botania:livingrock>, <bewitchment:focal_chalk>, <botania:livingrock>], [<bloodmagic:slate:1>, <thaumcraft:salis_mundus>, <bloodmagic:slate:1>], [<botania:livingrock>, <botania:blazeblock>, <botania:livingrock>]]);
 recipes.remove(<botania:conjurationcatalyst>);
-recipes.addShaped(<botania:conjurationcatalyst>, [[<ore:livingrock>, <bewitchment:sigil_purity>, <ore:livingrock>], [<botania:pylon:1>, <botania:alchemycatalyst>, <botania:pylon:1>], [<ore:livingrock>, <bewitchment:sigil_mending>, <ore:livingrock>]]);
```

Comment:
Recipe works without problem - therefore no change is needed - make sure you are using both Sigils Mending and Purity.